### PR TITLE
Open sessions

### PR DESCRIPTION
**Pull request by j-jasik**
THIS IS NOT COMPLETE. It is laying the groundwork for future work on open sessions, but before I can do that we need some work on the "roles and permissions" side of things.

### CODE REVIEW

File: sciencelabs/session/__init__.py [hunk 1]
```diff
@@ -234,7 +257,11 @@ def add_tutor_submit(self):
             session_id = form.get('session-id')
             tutor_id = form.get('choose-tutor')
             time_in = form.get('time-in')
```
Comment (c-schwarze) [inline sciencelabs/session/__init__.py]: for simple checks like these, you could do: `time_in = form.get('time-in') or None`

File: sciencelabs/session/__init__.py [hunk 2]
```diff
@@ -276,3 +311,43 @@ def create_session_submit(self):
...
+            set_alert('danger', 'Failed to close session: ' + str(error))
+            return redirect(url_for('SessionView:close_open_session', session_id=session_id))
+
+    def restore_deleted_session(self, session_id):
+        try:
+            self.session.restore_deleted_session(session_id)
+            set_alert('success', 'Session restored successfully!')
+            return redirect(url_for('SessionView:index'))
+        except Exception as error:
+            set_alert('danger', 'Failed to restore session: ' + str(error))
```
Comment (c-schwarze) [inline sciencelabs/session/__init__.py]: 👍 the alerts are setup nicely.

File: sciencelabs/session/__init__.py (hunk 1 above)
Comment (j-jasik) [reply sciencelabs/session/__init__.py]: Oh I didn't know that! I'll change that right away.

File: sciencelabs/session/__init__.py (hunk 2 above)
Comment (j-jasik) [reply sciencelabs/session/__init__.py]: 😄

**Pull request merged**